src/main/java/org/folio/dataexp/service/DataExportAllService.java

```diff
@@ -50,8 +50,12 @@ private UUID getJobProfileId(ExportAllRequest exportAllRequest) {
 
   private UUID getDefaultJobProfileId(ExportAllRequest exportAllRequest) {
     if (nonNull(exportAllRequest.getIdType())) {
-      return jobProfileEntityRepository.findIdOfDefaultJobProfileByName(exportAllRequest.getIdType().getValue()).get(0);
+      return switch (exportAllRequest.getIdType()) {
+        case INSTANCE -> UUID.fromString(DEFAULT_INSTANCE_JOB_PROFILE_ID);
+        case HOLDING -> UUID.fromString(DEFAULT_HOLDINGS_JOB_PROFILE_ID);
+        case AUTHORITY -> UUID.fromString(DEFAULT_AUTHORITY_JOB_PROFILE_ID);
+      };
     }
-    return jobProfileEntityRepository.findIdOfDefaultJobProfileByName(ExportRequest.IdTypeEnum.INSTANCE.getValue()).get(0);
```

Comment:
Hi @obozhko-folio ,  is it possible to remove JobProfileEntityRepository itsef or at least its findIdOfDefaultJobProfileByName method

Reply:
Done